packages/core/src/test/shared/sam/sync.test.ts

```diff
@@ -1030,6 +1030,91 @@ describe('SAM runSync', () => {
             prompterTester.assertCallAll()
         })
 
+        it('[entry: command palette] specify and save flag (with --watch) should save params before starting SAM process', async () => {
```

Comment:
95% of the 100 lines in this test are identical to the test above it. The `Copy-Paste Detection` CI job warned about this. This is not acceptable in this codebase.